code/modules/mining/ores_coins.dm

```diff
@@ -263,72 +266,78 @@
 	if(cmineral)
 		name = "[cmineral] coin"
 
+/obj/item/weapon/coin/examine(mob/user)
+	..()
+	if(value)
+		user << "<span class='info'>It's worth [value] credit\s.</span>"

```

Comment:
should add money goggles that let you see the current market value of shit when you examine them